database/scripts/01_00_00/05/ddl/tables/MODIFY.PUBLISHED_INCIDENT_DETAIL.sql

```diff
@@ -0,0 +1,7 @@
+-- Add FIre Centre Name
+ALTER TABLE "wfnews"."published_incident_detail" ADD COLUMN IF NOT EXISTS "fire_centre_name" VARCHAR(120);
+COMMENT ON COLUMN "wfnews"."published_incident_detail"."fire_centre_name" IS 'fire_centre_name is the name of fire centre of the incident'
```

Comment:
The comment on the existing column `"wfnews"."published_incident_detail.fire_centre"` should be updated to reflect that it's only storing the Code. Additionally, now that we have a fire_centre_name, column, that old column should be renamed to fire_centre_code or something sensible to reflect what it now is.